.github/workflows/ci.yml

```
@@ -15,35 +15,37 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [23.x, 22.x, 20.x]
-        # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
+				node:
```

```suggestion
				node:
```

Here strange identation